pro/src/pages/Home/StatisticsDashboard/__specs__/StatisticsDashboard.spec.tsx

```diff
@@ -115,4 +123,26 @@ describe('StatisticsDashboard', () => {
 
     expect(screen.getByText('Vos offres publiées')).toBeInTheDocument()
   })
+
+  describe('Create offer button displaying', () => {
```

Comment:
(suggestion) en général je préfère ne pas ajouter de describe intermédiaire, parce que sémantiquement describe est utilisé pour dire "on teste ce bout de code là" comme un composant ou une fonction

Et en plus les noms de tests dans les it juste en dessous sont déjà autoportants donc pas de raison de rajouter du code en plus :smile: